PocketKit/Sources/Sync/ApolloClientProtocol+Extensions.swift

```diff
@@ -70,16 +72,17 @@ public extension ApolloClientProtocol {
         }
     }
 
-    private static func checkForServerError(_ error: Error) {
+    private static func handleServerError(_ error: Error) -> Bool {
 
         // Codes we wish to notify the user about
-        let serverErrorCodes = [429]
+        let serverErrorCodes = [429, 500]
```

Comment:
```suggestion
        let serverErrorCodes = [429, 500, 503]
```
If you choose to add the 503 earlier.